pkg/cloud/vsphere/actuators/machine/actuator.go

```diff
@@ -78,6 +78,11 @@ func (a *Actuator) Create(
 		return err
 	}
 
+	if _, ok := machine.Annotations[constants.MaintenanceAnnotationLabel]; ok {
```

Comment:
Hi @sidharthsurana,

FWIW, all the `if` statements could be simplified as `if machine.Annotations[constants.MaintenanceAnnotationLabel] != ""` unless you envision using an empty value?

Reply:
I guess I don't ever see someone put an annotation with an empty value, but fair enough :)